src/messages.js

```diff
@@ -9,4 +9,5 @@ export default {
     "Click here to Login / Register<br />to get the mentor's details",
   CARD_ANONYMOUS_LIKE_TOOLTIP:
     'Click here to Login / Register<br />to add this mentor to favorites',
+  LOGOUT: 'Logout'
```

Comment:
The way I see it, the only purpose of `messages.js` is to provide a reference point for generic messages.
`CARD_APPLY_TOOLTIP`, `CARD_ANONYMOUS_LIKE_TOOLTIP` and `LOGOUT` only being reference once.

Reply:
Actually it meant to be the foundation of localization to support more languages. It's not adopted widely though :/
We'll do it soon.